DUI3-DX/Converters/Autocad/Speckle.Converters.Autocad2023.DependencyInjection/Speckle.Converters.Autocad2023.DependencyInjection.csproj

```diff
@@ -2,6 +2,7 @@
 
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
+        <PlatformTarget>x64</PlatformTarget>
```

Comment:
Was it needed?

Reply:
Because the autocad api is x64 only. If you ommit a `PlatformTarget` it defaults to `AnyCPU`. this will give you a msbuild warning saying there is a mismatch u
between processor architecture. (because you can't be compatible with any CPU if you depend on x64 dependencies)